### PR TITLE
Implementing acpype conversion of AMBER prmtop/crd files to GROMACS and associated changes

### PR DESCRIPTION
**Pull request by davidlmobley**
* Implement acpype conversion of AMBER prmtop/crd files to GROMACS format (in `utils.py`)
* Include `acpype.py` in package
* Adding D. Mobley authorship to `__init__.py`. 
* Implement acpype conversion testing in `utils/test_utils.py` (**not yet tested due to slowness in running tests**)
* Fix a documentation error in `run_tleap` within `utils.py`

### CODE REVIEW

**Comment by davidlmobley** [comment]
Er, so I'm new in dealing with these build tests. Can someone comment on what this means? (I clicked through and it wasn't totally obvious - language not set?)

**Comment by kyleabeauchamp** [comment]
You can't use the print statement in python 3.  You need to use print as a function.  E.g.

```
print("Hello World")
```

File: openmoltools/utils.py [hunk 1]
```diff
@@ -182,6 +185,77 @@ def run_tleap(molecule_name, gaff_mol2_filename, frcmod_filename, prmtop_filenam
...
+    #Define basename for output
+    basename = os.path.join( outdir, 'output')   
+
+ 
+    #Set up acpype
+    system = acpype.MolTopol( acFileXyz = in_crd, acFileTop = in_prmtop, basename = basename, is_sorted = is_sorted, gmx45 = True, disam = True )  
+
+    #Print debug info if desired
+    if debug: 
+        print system.printDebug('prmtop and inpcrd files parsed')
```
Comment (kyleabeauchamp) [inline openmoltools/utils.py]: This print needs to be a function for py3 compat.

**Comment by kyleabeauchamp** [comment]
LGTM

**Comment by davidlmobley** [comment]
@kyleabeauchamp - I think this is ready to merge once you (or someone else) takes a look at it to make sure it looks OK.

**Pull request merged**